cmd/argo/commands/common/get_test.go

```diff
@@ -41,7 +42,7 @@ func testPrintNodeImpl(t *testing.T, expected string, node wfv1.NodeStatus, getA
 		printNode(w, node, workflowName, "", getArgs, util.GetPodNameVersion())
 	}
 	err := w.Flush()
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
there are _some_ cases where an `assert.NoError` and continuing after could be fine, i.e. to give more details about the error with further assertions, but I think the vast majority (if not all) of the test usage is not intended that way